### PR TITLE
Fix upgrade db node print output

### PR DESCRIPTION
**Pull request by fgelcer**
changed the command to upgrade db packages to print its output and added doc on how to use this setting to our README.rst

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2322,7 +2322,7 @@ def update_scylla_packages(node, queue):
             node.remoter.run('sudo yum update -y --skip-broken')
             # replace the packages
             node.remoter.run('yum list installed | grep scylla')
-            node.remoter.run('sudo rpm -URvh --replacefiles /tmp/scylla/* | true')
+            node.remoter.run('sudo rpm -URvh --replacefiles /tmp/scylla/*', ignore_status=True, verbose=True)
```
Comment (roydahan) [inline sdcm/cluster.py]: rethinking about it, why do we ignore status?
why did we have the "true" on first place...
I would expect that if the rpm install fails, the test will fail.

**Comment by amoskong** [comment]
On Tue, Jul 16, 2019 at 7:21 PM Roy Dahan <notifications@github.com> wrote:

> *@roydahan* commented on this pull request.
> ------------------------------
>
> In sdcm/cluster.py
> <https://github.com/scylladb/scylla-cluster-tests/pull/1161#discussion_r303854973>
> :
>
> > @@ -2322,7 +2322,7 @@ def update_scylla_packages(node, queue):
>              node.remoter.run('sudo yum update -y --skip-broken')
>              # replace the packages
>              node.remoter.run('yum list installed | grep scylla')
> -            node.remoter.run('sudo rpm -URvh --replacefiles /tmp/scylla/* | true')
> +            node.remoter.run('sudo rpm -URvh --replacefiles /tmp/scylla/*', ignore_status=True, verbose=True)
>
> rethinking about it, why do we ignore status?
>
why did we have the "true" on first place...
>
Sometimes, the new package is successfully installed, but rpm returned
non-zero for conflicts or something.

> I would expect that if the rpm install fails, the test will fail.
>
We can try to install the new packages by `sudo yum install  /tmp/scylla/*`,
it's smarter than the rpm command.

**Comment by fgelcer** [comment]
yum is obviously a smarter option than rpm commands, but it might demand a better testing of this feature, while this PR is for improving the messages while running the upgrade.

File: sdcm/cluster.py (hunk 1 above)
Comment (fruch) [reply sdcm/cluster.py]: yeah, seem like that if we are using this feature, it's important to use that this one should work.
yeah we should change that

**Comment by fruch** [comment]
also while @fgelcer was using this, we found an issue with the `seed_nodes = self.get_seed_nodes()` call causing some nodes to not upgrade if the seed node isn't the first node.

**Comment by fruch** [comment]
Also I've asked @fgelcer to open this PR mainly for documenting how to use this feature.
we can take some of the fixes to a new PR/Issue

**Comment by fgelcer** [comment]
Actually it did upgrade, it just didn't stop before

File: sdcm/cluster.py (hunk 1 above)
Comment (bentsi) [reply sdcm/cluster.py]: @fgelcer please change `ignore_status` to `False`

Comment (fgelcer) [reply sdcm/cluster.py]: @bentsi , done

**Comment by fruch** [comment]
@bentsi anything holding this one ? seems like good enough to merge... mostly docs

**Comment by bentsi** [comment]
@fgelcer you didn't put backport labels

**Comment by fgelcer** [comment]
> @fgelcer you didn't put backport labels

@roydahan , just to be sure, do we want to backport it to 3.1 and 2019.1?

**Pull request merged**